checklists/alz_checklist.en.json

```diff
@@ -564,12 +564,12 @@
         {
             "category": "Network Topology and Connectivity",
             "subcategory": "App delivery",
-            "text": "If you use customer-managed TLS certificates with Azure Front Door, use the 'Latest' certificate version. Reduce the risk of outages caused by manual certificate renewal.",
+            "text": "Develop a plan for delivering application content from your Workload spokes, using the Application Delivery checklist as a guide.",
```

Comment:
Are we able to change the wording from "Application Delivery checklist" to recommendations from the link? IE. Leverage App GW / AFG for secure delivery of web applications. Maybe we can split into multiple checklist items

Reply:
More detailed and in-depth application delivery checks can go in separate list; however, we still want to capture some high-level recommendations as part of the ALZ checklist to align to our CAF guidance.  We can add a few items to capture the recommendations from:

https://learn.microsoft.com/en-us/azure/cloud-adoption-framework/ready/azure-best-practices/plan-for-app-delivery

Reply:
Updated the text, let me know if that work!